packages/@lwc/ssr-compiler/src/compile-template/transformers/component/slotted-content.ts

```diff
@@ -199,6 +201,8 @@ export function getSlottedContent(
         lightSlotContent.length > 0 || scopedSlotContent.length > 0
     );
 
+    cxt.isSlotted = false;
+
```

Comment:
What if `ctx.isSlotted` is already true on line 171? If we have two nested slotted contents in a row, then we want to reset back to true, not false.

Here is an example test case that fails due to this: 80a568e57

Reply:
@nolanlawson thanks for the test case! I added it and pushed a simple fix.